internal/node/hybrid/hybrid.go

```diff
@@ -105,28 +105,30 @@ func (hnp *HybridNodeProvider) Logger() *zap.Logger {
 }
 
 func (hnp *HybridNodeProvider) Validate() error {
+	// Create a background context for API server endpoint resolution validation
+	ctx := context.Background()
```

Comment:
Validate is run from the `Initer` struct's `Run` method, which takes the following signature:
```
func (i *Initer) Run(ctx context.Context) error {
```
Can we just use this context instead of creating a new one?